modules/Hoot/tools/grailPull.js

```diff
@@ -200,6 +201,16 @@ export default class GrailPull {
         } );
     }
 
+    // Checks if there is tm project and task info set. Also check that the custom
+    static customDataExtentEnabled() {
+        if ( sessionStorage.getItem('tm:project') && sessionStorage.getItem('tm:task') &&
+            Hoot.context.layers().layer('data').enabled() && Hoot.context.layers().layer('data').hasData() ) {
```

Comment:
This line is checking the presence of gpx task bounds data?

Reply:
yea and that its enabled just incase someone has a bounds but they dont want to use it for a pull, they can still disabled the bounds